core/domain/stats_jobs_one_off.py

```diff
@@ -1407,18 +1407,26 @@ def map(item):
                     playthrough_issue_registry.Registry.get_issue_by_type(
                         issue['issue_type']).customization_arg_specs)
                 all_ca_names = set([ca_spec.name for ca_spec in ca_specs])
-
                 yield (
                     'ExplorationIssue',
```

Comment:
Don't do it like this. Instead, yield ('ExplorationIssue -- SUCCESS', 1) and ('ExplorationIssue -- FAILURE', '%s %s'.encode('utf-8') % (item.exp_id, issue['issue_type'])). Maybe include even more info in the failure case, e.g. the issue or playthrough ID.

Ditto below.

Then, in reduce(), if the key ends with 'SUCCESS', yield the length of the results, else yield the individual results. See other jobs for examples of this pattern.

(Btw -- I don't think you filed a run request for this job?)